index.html

```diff
@@ -16,12 +16,69 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="purple">
     <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h4 class="code-journal">Code Journal</h4>
+        </div>
+      </div>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <div data-view="entry-form">
+      <form class="journal-form">
+
+        <div class="row">
+
+          <div class="column-half">
+            <div class="form-box">
+              <h2>New Entry</h2>
+              <div class="image">
+                <img class="placeholder"
+     src="./images/placeholder-image-square.jpg"
+     alt="Photo URL Image">
+              </div>
+            </div>
+          </div>
+
+          <div class="column-half">
+
+            <div class="form-box">
+              <div class="title-input">
+                <p>Title</p>
+                <input required class="input-text" name="title" type="text" size="15"/>
+                <p>Photo URL</p>
+                <input required class="input-text" name="url" type="url" size="15"
+  id="photo-url"  />
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+
+          <div class="column-full">
+            <div class="bottom-section">
+              <div class="form-box">
+                <div class="notes-section">
+                  <p>Notes</p>
```

Comment:
This should be a `label`. Make sure you configure it correctly, I recommend reviewing `html-forms`